util/rbac/rbac_test.go

```diff
@@ -371,6 +371,8 @@ func TestValidatePolicy(t *testing.T) {
 	goodPolicies := []string{
 		"p, role:admin, projects, delete, *, allow",
 		"",
+		"#",
+		`p, "role,admin", projects, delete, *, allow`,
```

Comment:
We should also test for strings starting with whitespace.

Reply:
done